src/param.rs

```diff
@@ -116,6 +117,9 @@ pub enum Param {
 
     /// For QR
     GroupName = b'g',
+
+    /// For MDN-sending job
+    MessageId = b'I',
```

Comment:
This needs to be `MsgId` -- MessageId typically denotes the rfc724 one, and MsgId, msg_id is the int we use.